README.md

```diff
@@ -1,3 +1,114 @@
-# FlexyIgnitionDemo
-A Java Application for Flexy, providing a Multi-Dashboard Demo on the Ignition platform from Inductive Automation
+# HMS Ewon Ignition Integration Demo #
 
+----------
+
+## About ##
+HMS Ewon Ignition Integration Demo is a small suite of applications that showcase the features of integration between HMS Industrial Network's Ewon platform, and the Inductive Automation Ignition platform.
+
+HMS Ewon Ignition Integration Demo was developed by the HMS Solution Center using the Ewon Java Library. For more information on this demo and developing applications with the Ewon Java Library, please contact us at [us-services@hms.se](mailto:us-services@hms.se "us-services@hms.se") and/or visit the Ewon Developer website at [https://developer.ewon.biz/](https://developer.ewon.biz/ "https://developer.ewon.biz/")
```

Comment:
There are multiple solution centers so you could just say developed by HMS or state that its the North America's Solution Center or whatever the current official name is.